Task03-OnePieceWebsite/home-page.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * All Rights Reserved
+ * This software is property information of AKUREY S.A.
+ * Contact us at contact@akurey.com
+ * @summary short description for the file
+ */
+ function showSipnosis() {
+    let moreText = document.getElementById("extra-sipnosis-description");
+    let btnText = document.getElementsByClassName("read-all-label")[0];
+    let icon = document.getElementsByClassName("material-icons")[0];
+    let logo = document.getElementsByClassName("logo-header")[0];
```

Comment:
I will use
`document.querySelector('.reall-all-label')` for example, this will get the first match on the document, the same for the following

Reply:
Should I also replace the .getElementById with the .querySelector?

Reply:
If you want to, the change was for getting elements by class name, because it will return the array and then you have to get the first element, so the query selector is more direct because it will give you the first match. querySelectorAll will give all coincidences into an array just in case.
So for an id, it's your choice if you want to use the query or id selector. 😁

Reply:
Ok ok thanks!